book.tex

```diff
@@ -1403,10 +1405,8 @@ \subsubsection{div Attribute}
 IN$: 
 \end{ffcode}
 
-
-
 \subsection{Additional Attributes For Working With int Objects}
-The object \ff{org.eolang.math.number} is a decorator of \ff{org.eolang.int}. So the number object contains methods for working with \ff{int} data type objects. Before using these methods, you need to make such an application:
+The object \ff{org.eolang.math.number} is a decorator of \ff{org.eolang.int}. So the number object contains methods for working with \ff{int} data objects. Before using these methods, you need to make such an application:
```

Comment:
@c71n93 As far as I know `number` object also decorates `float` object

Reply:
@Graur I check implementation of `number`. I think this object more like a wrapper for `float` and `int` than a decorator. Is that right?

Reply:
@c71n93 Accoring to p.9 (Math) of https://arxiv.org/pdf/2206.02585.pdf paper it's a decorator.

> The object number is a decorator of int and float
with the following additional attributes:

Reply:
@Graur But [here](https://github.com/objectionary/home/blob/master/objects/org/eolang/math/number.eo) number don't decorates `int` and `float`. It just wrap this objects and provide some attributes to work with them.

Reply:
@c71n93 Indeed. Looks like a bug. We should fix it. Can you create an issue in `eo-math`?

Reply:
@Graur Okay. But didn't you mean `home`? Because `number` object located in `home` repo.

Reply:
> @Graur Okay. But didn't you mean `home`? Because `number` object located in `home` repo.

@Graur Actually there is `number` object in `eo-math` too. And [here](https://github.com/objectionary/eo-math/blob/master/src/main/eo/org/eolang/math/number.eo) it really decorates object that applied.

Reply:
@c71n93 It's a combination of dictionary and object factory, called Objectionary. Inspired by idea from the book [Object Thinking](http://amzn.to/266oJr4) by David West.
You can read more about it here: https://www.yegor256.com/2021/10/21/objectionary.html

Long story short, `home` repository stores last released `.eo` files from others `objectionary` repositories, including `eo-math`. That's why you can find different versions of the same object. All EO compilations getting `.eo` files from `home` repository.

Reply:
@Graur Okay, thanks. So we don't need to fix it in `home`?